test/cli/seed-make.spec.js

```diff
@@ -378,4 +379,75 @@ development: {
       );
     });
   });
+
+  describe('--timestamp-filename-prefix option: make seed with timestamp filename prefix', () => {
+    /**
+     * @type FileTestHelper
+     */
+    let fileHelper;
+    beforeEach(() => {
+      fileHelper = setupFileHelper();
+    });
+
+    afterEach(() => {
+      fileHelper.cleanup();
+    });
+
+    before(() => {
+      process.env.KNEX_PATH = '../knex.js';
+    });
+
+    it('Creates a new seed using --timestamp-filename-prefix CLI flag', async () => {
+      const testFileName = `somename-${yyyymmddhhmmss()}`;
+      const seedGlobPath = `${process.cwd()}/seeds/*_${testFileName}.js`;
+      fileHelper.registerGlobForCleanup(seedGlobPath);
+
+      await execCommand(
+        `${NODE} ${KNEX} seed:make ${testFileName} --timestamp-filename-prefix --knexpath=../knex.js`,
+        {
+          expectedOutput: 'Created seed file',
+        }
+      );
+
+      const fileCount = fileHelper.fileGlobExists(seedGlobPath);
+
+      expect(fileCount).to.equal(1);
+    });
+
+    it('Creates a new seed using timestampFilenamePrefix parameter in knexfile', async () => {
+      const seedsDirectory = `${process.cwd()}/seeds`;
+      const testFileName = `somename-${yyyymmddhhmmss()}`;
+      const seedGlobPath = `${seedsDirectory}/*_${testFileName}.js`;
+      fileHelper.registerGlobForCleanup(seedGlobPath);
+
+      const knexfileContents = `
+        module.exports = {
+          client: 'sqlite3',
+          connection: {
+            filename: __dirname + '/test/jake-util/test.sqlite3',
+          },
+          seeds: {
+            directory: '${seedsDirectory}',
+            timestampFilenamePrefix: true
+          },
+        };`;
+
+      fileHelper.createFile(
+        path.join(process.cwd(), 'knexfile.js'),
+        knexfileContents,
+        { isPathAbsolute: true }
+      );
+
+      await execCommand(
+        `${NODE} ${KNEX} seed:make ${testFileName} --knexpath=../knex.js`,
+        {
+          expectedOutput: 'Created seed file',
+        }
+      );
+
+      const fileCount = fileHelper.fileGlobExists(seedGlobPath);
```

Comment:
Isn't this going to be flaky due to slight mismatch of time between setting testFileName and generating actual stub? Wonder if we could mock timer to make this test deterministic or use less strict glob that ignores seconds part.

Reply:
I put a timestamp in the end of filename to eliminate chances of create files with same name between tests. I not testing that this timestamp value will be the same that the new option generates. But I will change these tests to more clearly logic.